css/style.scss

```diff
@@ -755,10 +755,11 @@ input.input-inline {
 
 	thead td {
 		opacity: .5;
+		padding: 15px;
 	}
 
 	td {
-		padding: 0 10px;
+		padding: 3px 10px;
```

Comment:
@juliushaertl please revert this line to `padding: 0 10px;`

If you think rows must be 50px high, then you can increase this to 25px: `#boardlist .icon { padding: 25px; }`

The difference between these 2 solutions is that with the latter, buttons fill full height of the row.

P.S. In my opinion, 44px is enough for row height anyway.